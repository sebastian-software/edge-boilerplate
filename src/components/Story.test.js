import initStoryshots from "@storybook/addon-storyshots"

initStoryshots({
  configPath: "./__storybook__"
})
