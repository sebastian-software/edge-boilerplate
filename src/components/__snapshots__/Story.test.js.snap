// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Navigation basic demo 1`] = `
<nav>
  <ul>
    <li>
      <a
        aria-current="true"
        className="active"
        href="/"
        onClick={[Function]}
        style={Object {}}
      >
        Home
      </a>
    </li>
    <li>
      <a
        aria-current={false}
        className={undefined}
        href="/counter"
        onClick={[Function]}
        style={undefined}
      >
        Counter
      </a>
    </li>
    <li>
      <a
        aria-current={false}
        className={undefined}
        href="/localization"
        onClick={[Function]}
        style={undefined}
      >
        Localization
      </a>
    </li>
  </ul>
</nav>
`;
